src/observers/TelemetryObserver.ts

```diff
@@ -44,6 +44,13 @@ export class TelemetryObserver {
                 let telemetryEvent = <TelemetryEvent>event;
                 this.reporter.sendTelemetryEvent(telemetryEvent.eventName, telemetryEvent.properties, telemetryEvent.measures);
                 break;
+            case EventType.ProjectConfigurationReceived:
+                let projectConfig = (<ProjectConfiguration>event).projectConfiguration;
+                telemetryProps['ProjectFilePath'] = projectConfig.ProjectFilePath;
```

Comment:
Is it possible for us to make this a propertybag/object and just copy all the members?